docs/api-readme.md

```diff
@@ -0,0 +1,35 @@
+# LitElement API Documentation
+
+## Install @polymer/lit-element
+
+```
+npm install --save @polymer/lit-element
+```
+
+## Modules
+
+### lit-element
+
+[lit-element](/api/modules/_lit_element_.html)
```

Comment:
The "lit-element" header, and then the "lit-element" link make the page look visually repetitive
<img width="810" alt="screen shot 2018-12-21 at 10 32 05 am" src="https://user-images.githubusercontent.com/522948/50357698-b589d900-050b-11e9-8eeb-44217fdd3d76.png">

Can we either put the link in the header, or add some text to the link like "lit-element API"? Same below.

Reply:
The whole api docs output is visually and otherwise repetitive so it's in theme ;) jk, I'll fix it